commons/commons-api/src/test/java/fr/gouv/vitamui/commons/api/utils/MetadataSearchCriteriaUtilsTest.java

```diff
@@ -65,10 +67,35 @@ public void handleSimpleFieldCriteria() throws InvalidCreateOperationException {
             .setValues(List.of(new CriteriaValue().setValue("2023-03-05T23:00:00.000Z")))
             .setDataType(ArchiveSearchConsts.CriteriaDataType.DATE.name());
 
-        MetadataSearchCriteriaUtils.handleSimpleFieldCriteria(queryToFill, searchCriteria);
+        MetadataSearchCriteriaUtils.fillQueryFromCriteriaList(queryToFill, Collections.singletonList(searchCriteria));
+
+        Assertions.assertEquals(
+            "{\"$or\":[{\"$and\":[{\"$or\":[{\"$gt\":{\"ActivationDate\":\"2023-03-06\"}}]}]}]}",
```

Comment:
Est-ce normal d'avoir or -> and -> or -> gt ?

Reply:
c'est la requête générée par vitam ui, je n'ai pas touché à cette partie... honnêtement, pour bien faire, il faudrait revoir toute cette partie, le code est particulièrement alambiqué